unittests/DSL/TSBuilderTests.cs

```diff
@@ -180,5 +180,36 @@ public void InsertNewLine()
             builder.InsertNewLine(0);
             Assert.Equal("\n", builder.ToString());
         }
+
+        [Fact]
+        public void AddPropertyToParentObjectWhileBuildingChildObject()
+        {
+            TSBuilder builder = new TSBuilder();
+            Exception exception = null;
```

Comment:
Consider using the `Exception exception = Assert.Throws(() => builder.Object(...))` pattern

Reply:
I didn't know that Assert.Throws() returned the validated exception. I'll change it to that.